README.md

```diff
@@ -123,7 +123,7 @@ To make further requests, check out the documentation - this gem should follow t
 ## Contributing to rubypress
 
 Pull requests welcome.
- 
+
```

Comment:
Sure, I use Atom too. I'll ignore this for now but I'd advise you to watch out for projects that are more strict on commits!